### PR TITLE
add test case for  DataSourceConnectionUrlUtil

### LINKED ISSUE #7735: Add unit tests for DataSourceConnectionUrlUtil
## Introduction
`DataSourceConnectionUrlUtil` is to assemble configuration items, e.g., password, database name, to get a complete and correct JDBC URL for the connection pool.

## Task
1. Create unit test class for `DataSourceConnectionUrlUtil`
2. Add assertion for `getUrl()` with `PostgreSQL` database input
3. Add assertion for `getUrl()` with `MySQL` database input
4. Raise a PR

## Tips
1. `DataNodeUtilTest` is a good example for your information
2. [Contributor guidance](https://shardingsphere.apache.org/community/cn/contribute/contributor/) is for the newbie. :-)

### PR DESCRIPTION
**Pull request by cenyu00**
Fixes #7735.

Changes proposed in this pull request:
- add test class DataSourceConnectionUrlUtilTest
-
-

### CODE REVIEW

File: shardingsphere-rdl-parser/shardingsphere-rdl-parser-binder/src/test/java/org/apache/shardingsphere/rdl/parser/binder/util/DataSourceConnectionUrlUtilTest.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    public void assertPostgreSQLGetUrl() {
+        DataSourceConnectionSegment segment = new DataSourceConnectionSegment();
+        segment.setHostName("127.0.0.1");
+        segment.setDb("test");
+        segment.setUser("root");
+        segment.setPort("3306");
+
+        PostgreSQLDatabaseType databaseType = new PostgreSQLDatabaseType();
+        String actual = DataSourceConnectionUrlUtil.getUrl(segment, databaseType);
+        String expected = String.format("%s//%s:%s/%s", "jdbc:postgresql:",
```
Comment (tristaZero) [inline shardingsphere-rdl-parser/shardingsphere-rdl-parser-binder/src/test/java/org/apache/shardingsphere/rdl/parser/binder/util/DataSourceConnectionUrlUtilTest.java:57]: I think the `expected` is supposed to be an actual string value like `jdbc:mysql//xxxxxxxxxxxx`. As your expected value is the same as the expression in `DataSourceConnectionUrlUtil `, the assertion will always be true.

File: shardingsphere-rdl-parser/shardingsphere-rdl-parser-binder/src/test/java/org/apache/shardingsphere/rdl/parser/binder/util/DataSourceConnectionUrlUtilTest.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    }
+
+    @Test
+    public void assertPostgreSQLGetUrl() {
+        DataSourceConnectionSegment segment = new DataSourceConnectionSegment();
+        segment.setHostName("127.0.0.1");
+        segment.setDb("test");
+        segment.setUser("root");
+        segment.setPort("3306");
+
```
Comment (tristaZero) [inline shardingsphere-rdl-parser/shardingsphere-rdl-parser-binder/src/test/java/org/apache/shardingsphere/rdl/parser/binder/util/DataSourceConnectionUrlUtilTest.java:54]: Please remove the redundant blank line.

**Review by tristaZero — CHANGES_REQUESTED** [review]
Hi @cenyu00 welcome your first PR, and I wait it for a long time. :-）
Could you look at my comments?

**Comment by cenyu00** [comment]
@tristaZero ,Thank you.
Due to insufficient understanding of the project, it takes a long time.
I feel that the ss community is friendly to newcomers and the pursuit of high-quality code.
I will do more for the ss community.

**Review by tristaZero — APPROVED** [review]
Thanks @cenyu00, It looks better now. 
Welcome your more interaction with this open-source community, and enjoy it!

**Comment by tristaZero** [comment]
@cenyu00 BTW, would you like to continue with #7849, which is a subsequent task for this PR? If you want, please comment on that issue there, and I will assign it to you. Otherwise, please let me know here, and I will give it to other volunteer guys. :)

**Pull request merged**